example/hatchify-app/frontend/App.tsx

```diff
@@ -11,12 +11,14 @@ export const hatchedReact = hatchifyReact(
   createJsonapiClient("http://localhost:3000/api", schemas),
 )
 
-const TodoList = hatchedReact.components.Todo.Collection
-const TodoColumn = hatchedReact.components.Todo.Column
-const TodoEmptyList = hatchedReact.components.Todo.Empty
-// hatchedReact.model.Todo.findAll({}).then(([todos]) => {
+const TodoList = hatchedReact.components.Admin_Todo.Collection
```

Comment:
we probably don't want namespaces in this `example/hatchify-app`, it's fine to use for local testing but what we end up moving to main should be non-namespaced since this example repo follows the getting started guide (which uses sqlite and can't support namespaces)

might be worth making a future `example/hatchify-app-namespaces` that does the same but runs postgres + namespace

Reply:
^ not as part of this PR, just if we plan on supporting namespaces beyond 1 client